content/tutorials/permissionless-paymaster/10.index.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Integrate permissionless multi-signer paymaster into your Dapp
+description: Permissionless paymaster allows multiple Dapps to seamlessly sponsor gas for their users using signature verification.
+---
+
+This tutorial shows you how to integrate this paymaster in your Dapp.
+
+You will:
+
+- Create a signer address that will sign paymaster data for gas sponsorship.
+- Deposit gas funds and add the signer address.
+- Sign EIP-712 standard signature on the paymaster data by the signer.
+- Send the transaction with paymaster data & signature.
+
+## Prerequisites
+
+- Make sure you have a basic front-end integration ready, you can create a standard frontend [using zksync-cli](https://github.com/matter-labs/era-compiler-solidity/tree/main#system-requirements).
```

Comment:
Is this expected link for the cli?

Reply:
It's a mistake. Have updated with the correct link, ty!